tests/e2e.sh

```diff
@@ -37,12 +36,12 @@ function error_exit() {
     exit ${2:-1}
 }
 
-. ${ROOT}/istio.VERSION || error_exit "Could not source versions"
```

Comment:
Why are you removing this ?

Reply:
We are not using istio.Version, are we? Versions will all be handled from the flag in e2e-suite.sh, do I missing something? If we are not updating istio.VERSION anymore, I think it would be better to stop people from replying on it implicitly.

Reply:
I'm very unclear on whether it is used or not

if not : where is the proxy sha coming from ?

Reply:
Leave it for now, and have someone else remove it when we get rid of istio.VERSION.